Casks/a/around.rb

```diff
@@ -11,7 +11,7 @@
 
   livecheck do
     url "https://downloads.around.co/latest.yml"
```

Comment:
```suggestion
    url "https://downloads.around.co/latest-mac.yml"
```

We need `latest-mac.yml`, as `latest.yml` contains Windows files.